pyat/at/load/utils.py

```diff
@@ -150,7 +159,14 @@ def err_message(message, *args):
         if pass_method is not None:
             pass_to_class = PASS_MAPPING.get(pass_method)
             length = float(kwargs.get('Length', 0.0))
-            if (pass_method == 'IdentityPass') and (length != 0.0):
+            extension = sysconfig.get_config_vars().get('EXT_SUFFIX', '.so')
+            file_path = os.path.realpath(os.path.join(__file__,
+                                                      '../../integrators',
+                                                      pass_method + extension))
+            if not os.path.exists(file_path):
+                raise AttributeError('PassMethod {0} does not exist.'
```

Comment:
We should report the element index (if available) in the error message, like it is done in `err_message`

Reply:
will do

Reply:
I've updated `err_message` so that it can be used here as well.